tools/autograd/derivatives.yaml

```diff
@@ -404,6 +404,10 @@
   self: grad * tensor
   tensor: grad * self
 
+- name: vdot(Tensor self, Tensor other) -> Tensor
```

Comment:
Does this mean that vdot, even when it defers to dot, is also not differentiable, or will that work?

Reply:
Yes `vdot` wouldn't be differentiable even when it will defer to `dot`, which I feel is better than `vdot` being differentiable for real types and not for complex.

Reply:
Sounds good. Just wanted to be sure we're clear on the behavior. Thanks for being so thorough, @kshitij12345.